frontend/app/electron/main/index.ts

```diff
@@ -71,17 +71,42 @@ async function onReady(): Promise<void> {
 
   getWindow().webContents.on('before-input-event', (event, input) => {
```

Comment:
What does event look like here? does it have a target?

Reply:
Great, if you can make it run only on the valid key combo that would be great, thanks.